libs/libbuiltin/libgcc/Make.defs

```diff
@@ -24,6 +24,12 @@ endif
 
 ifeq ($(CONFIG_COVERAGE_MINI)$(CONFIG_ARCH_TOOLCHAIN_GCC),yy)
 CSRCS += gcov.c
+
+GCOV_CFLAGS += -fno-profile-arcs -fno-test-coverage
+
+bin/gcov.o: CFLAGS += $(GCOV_CFLAGS)
```

Comment:
bin/gcov.o_CFLAGS += (GCOV_CFLAGS)
kbin/gcov.o_CFLAGS += $(GCOV_CFLAGS)

Reply:
Adding CFLAGS to gcov.c is bin/gcov.o: CFLAGS += xxx? bin/gcov.o_CFLAGS += (GCOV_CFLAGS) will not take effect